UnitsNet.Tests/CustomCode/ForceTests.cs

```diff
@@ -49,13 +49,34 @@ public void ForceDividedByAreaEqualsPressure()
         }
 
         [Test]
+        public void PressureByAreaEqualsForce()
+        {
+            Force force = Force.FromPressureByArea(Pressure.FromNewtonsPerSquareMeter(9), Area.FromSquareMeters(9));
```

Comment:
I would prefer using different numbers as arguments, since in theory you are open to bugs where only one of the arguments are used. It's highly theoretical, but I'd still change it just for the better practice.